core/templates/pages/contributor-dashboard-admin-page/contributor-admin-dashboard-page.component.ts

```diff
@@ -189,6 +190,18 @@ export class ContributorAdminDashboardPageComponent implements OnInit {
       );
   }
 
+  putEnglishLanguageNameAtFront(language: string): string {
```

Comment:
This might be better as a private method, since this functionality is only used within this file. Then, this method doesn't have to be tested and instead, we can focus on testing that `this.languageChoices` is initialized as desired.

Reply:
Done.